sample_design_and_estimation/estimation/winsorisation/method_specification.md

```diff
@@ -0,0 +1,338 @@
+$j$. These values are inputs to the calculation of the _design weight_ that is
+passed to Winsorisation.
+
+**$w_i$** is the _weight_ which, for business $i$, is calculated as:
+$w_i = a_i$ when **expansion estimation** is used
+$w_i = a_i g_i$ when **ratio estimation** is used
+
+**$L$** is the Winsorisation parameter supplied to the method - in ONS this is
+calculated from historic data using a bespoke method based on the survey
+requirements. Calculation of the L value is
+beyond the scope of this specification.
+
+**$a_i$** is the _design weight_ that is supplied to Winsorisation. The calculation
+ of this is outside the scope of the
+Winsorisation method itself.
+
+**$g_i$** is the _calibration factor_ that is supplied to Winsorisation. The
+calculation of this is outside the scope of the Winsorisation method itself.
+
+**${\overline{y}_h}$** is the
+_mean target value_, the sample mean for
+stratum $h$ for the period to be Winsorised
+
+$$\begin{equation}
+    \bar{y}_h = \text{mean target value}
+\end{equation}$$
+
+<!-- markdownlint-disable MD013 -->
+$$\begin{equation}
+    \text{mean target value} = \frac{\text{sum of target values}}{\text{number of target values}}
+\end{equation}$$
+<!-- markdownlint-enable MD013 -->
+
+$$\begin{equation}
+    \text{sum of target values}=\sum^{n_h}_{i=1}y_i
+\end{equation}$$
+
+$$\begin{equation}
+    \text{number of target values} = {n_h}
+\end{equation}$$
+
+**$x_i$** is the _auxiliary variable_ for a given sampled _target value_
+
+**${\widehat{\mu}}_{i}$** is the _predicted unit value_ used in **ratio estimation**
+and is calculated using the data to be Winsorised for each observation and the
+sum of the _weighted target values_ divided by the sum of the _weighted auxiliary
+values_ for each calibration group $j$ multiplied by the supplied auxiliary value
+
+$$\begin{equation}
+\hat{\mu}_i = \text{predicted unit value}
+\end{equation}$$
+
+<!-- markdownlint-disable MD013 -->
+$$\begin{equation}
+\text{predicted unit value} = x_i \frac{\text{sum of weighted target values}}{\text{sum of weighted auxiliary values}}
+\end{equation}$$
+<!-- markdownlint-enable MD013 -->
+
+$$\begin{equation}
+\text{sum of weighted target values} = \sum_{i=1}^{n_j} a_i y_i
+\end{equation}$$
+
+$$\begin{equation}
+\text{sum of weighted auxiliary values} = \sum_{i=1}^{n_j} a_i x_i
+\end{equation}$$
+
+### Winsorisation
+
+Once a target value has been identified for Winsorisation the outlier
+weight, **$o_i$** is calculated as:
+
+$$\begin{equation}
+    o_i =
+    \begin{cases}
+1 & \text{if $y_i$  is not an outlier}\\
+\frac{y_i^*}{y_i} & \text{if $y_i$ is an outlier}
+    \end{cases}
+ % for multiple cases you need to add \usepackage{amsmath}
+\end{equation}$$
+
+Where
+
+$$\begin{equation}
+y_i^* =
+\begin{cases}
+    y_i & \text{if $y_i$ is not an outlier, i.e. $y_i\leq k_i$}\\
+    \frac{1}{w_i} y_i+(1- \frac{1}{w_i})k_i & \text{if $y_i$ is an outlier,
+    i.e. $y_i>k_i$}
+\end{cases}
+\end{equation}$$
```

Comment:
Hi Ben - thanks for the review. The implementation in code does simplify the step. The algebra is driven by methodology to get across the theory behind the method, since MQD are the experts in this area, and it has take a bit of back and forth to get to consensus on the algebra, I'll leave as is if that is ok.